src/main/java/com/cleanroommc/groovyscript/compat/mods/tinkersconstruct/Alloying.java

```diff
@@ -59,6 +64,7 @@ public boolean removeByOutput(FluidStack output) {
         return false;
     }
 
+    @MethodDescription(example = @Example("fluid('cobalt')*2,fluid('ardite')*2"))
```

Comment:
missing lang key, `groovyscript.wiki.tconstruct.alloying.removeByInputs`. this is logged in the `groovy.log`